### PR TITLE
Status badges

### PR DESCRIPTION
**Pull request by aurium**
As done for `fn-object`, this badges will help to remember when to update the dependencies, for the ecosystem. :-)

Plus: Adjust the reader parenting for a better semantic.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -291,5 +293,12 @@ function toInt (v) {
...
-### MIT
+## License
+**MIT**
+
+[1]: https://travis-ci.org/tjmehta/dat-middleware.png
+[2]: https://travis-ci.org/tjmehta/dat-middleware
+[3]: https://david-dm.org/tjmehta/dat-middleware.png
+[4]: https://david-dm.org/tjmehta/dat-middleware
+[5]: https://david-dm.org/tjmehta/dat-middleware/dev-status.png
+[6]: https://david-dm.org/tjmehta/dat-middleware#info=devDependencies
```
Comment (bkendall) [inline README.md]: can you replace `[1-6]` with names to make them slightly more descriptive? should make it less of a mystery when one is looking at the badge line and wondering what the numbers are w/o going back and forth :)

**Comment by bkendall** [comment]
Thanks for these, and for cleaning up the readme's headers a bit. Just one little nit, but looks great!

File: README.md (hunk 1 above)
Comment (aurium) [inline README.md]: @bkendall, you are a MD master! I dont know that was possible. :-)

**Comment by aurium** [comment]
Thanks @bkendall :-)

**Comment by bkendall** [comment]
:+1: nice

**Pull request merged**